content/blog/esc-projects-environment-tags-launch/index.md

```diff
@@ -63,6 +63,13 @@ We offer numerous ways for you to get started with Projects via the Pulumi Web C
   pulumi/cloud-pe/demo
  ```
 
+### Breaking Changes
+
+With the introduction of Projects, all existing environments part of the `default` project will continue to work without requiring any changes. However, when moving existing environments out of the default project you may notice some differences that we have outlined below.
+
+* If your environment was interpolating the `context.currentEnvironment.name` or `context.rootEnvironment.name` from [Contextual information](/docs/esc/environments/#pulumi-contextual-information), this value will now include the project for any environments outside of the `default` project.
+* If you were using any of the [secrets providers with OIDC](/docs/esc/environments/#using-secrets-providers-and-oidc), the subject will be either `pulumi:environments:org:<organization name>:env:<project name>/<environment name>` if no `subjectAttributes` are specified or if the `currentEnvironment.name` subject attribute is specified it will now resolve to `currentEnvironment.name:<project name>/<environment name>`.
```

Comment:
```suggestion
* If you were using any of the [providers with OIDC](/docs/esc/environments/#using-secrets-providers-and-oidc), the subject will be either `pulumi:environments:org:<organization name>:env:<project name>/<environment name>` if no `subjectAttributes` are specified or if the `currentEnvironment.name` subject attribute is specified it will now resolve to `currentEnvironment.name:<project name>/<environment name>`.
```

Reply:
going to keep as secrets providers since that's how it's referenced in the concepts doc